tests/certification/bindings/cron/config.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: cronbindingconfig
+spec:
+  features:
```

Comment:
Did you intend to add something here?

Reply:
It's ok to keep it like this. Some other tests have this too.

Reply:
It's ok, but why keep it if it's not doing anything? 🤔

Reply:
It you remove this, Dapr will crash. Don't ask Me why.